static/js/compose_validate.js

```diff
@@ -600,9 +600,14 @@ function validate_private_message() {
     ) {
         // Unless we're composing to a bot
         compose_error.show(
-            $t_html({defaultMessage: "Private messages are disabled in this organization."}),
+            $t_html({
+                defaultMessage:
+                    "You are not allowed to send private messages in this organization.",
+            }),
             $("#private_message_recipient"),
         );
+
+        $(".empty_feed_notice p").html("");
```

Comment:
This seems not robust -- what if we have multiple paragraphs in that template?

We should add a CSS class on the element to modify here.

Reply:
Aso, probably it makes more sense to just have a conditional in the template? Not sure what this line is for.

Reply:
> This seems not robust -- what if we have multiple paragraphs in that template?
> 
> We should add a CSS class on the element to modify here.

Okay, I got your point. Although there is no direct <p> child present in the parent container, but as per your saying I have updated the previous code and added a class `private_message_para` to the p element. Please review it. Thanks!

Reply:
> Also, probably it makes more sense to just have a conditional in the template? Not sure what this line is for.

The data which is coming in the template does not specify that the private message is enabled or not ,  but for this if we put an external parameter which can confirms that private message is enabled or not will create a problem. And the problem is that the paragraph which we are trying to see empty in the pm window will already show that para empty in initial rendering, which we wants to show empty after hitting the enter or send key.




Like you can see here in the screenshot that the para is already showing empty before hitting the send/enter key, but as per the issue we wanted to get it after writing the message to someone whom we wants to message.
![conditional rendering  view](https://user-images.githubusercontent.com/75739890/165532461-6dd80adc-2759-4377-b865-ddfdefdf8deb.png)

Reply:
> This seems not robust -- what if we have multiple paragraphs in that template?
> 
> We should add a CSS class on the element to modify here.

@timabbott can I have review on this PR. Thanks!

Reply:
> Aso, probably it makes more sense to just have a conditional in the template? Not sure what this line is for.

@timabbott please review this PR . Thanks